sqlmesh/core/renderer.py

```diff
@@ -202,19 +202,54 @@ def _resolve_tables(
         if not snapshots and not table_mapping and not expand:
             return expression
 
+        from sqlmesh.core.snapshot import to_table_mapping
+
         expression = expression.copy()
         with _normalize_and_quote(expression, self._dialect) as expression:
-            return _resolve_tables(
-                expression,
-                snapshots=snapshots,
-                table_mapping=table_mapping,
-                expand=expand,
-                deployability_index=deployability_index,
-                start=start,
-                end=end,
-                execution_time=execution_time,
-                **kwargs,
-            )
+            snapshots = snapshots or {}
+            table_mapping = table_mapping or {}
+            mapping = {**to_table_mapping(snapshots.values(), deployability_index), **table_mapping}
+            expand = {d.normalize_model_name(name, dialect=self._dialect) for name in expand} | {
```

Comment:
Is this the only line that actually changed? Does it make sense to reduce the PR to just this relevant part?

Reply:
yes, this is the only line that should have changed